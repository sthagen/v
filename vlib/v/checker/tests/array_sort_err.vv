fn main() {
	mut arr := [3, 2, 1]
	arr.sort(a < b, a)
}
