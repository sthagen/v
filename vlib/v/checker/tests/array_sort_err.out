vlib/v/checker/tests/array_sort_err.vv:3:6: error: expected 0 or 1 argument, but got 2
    1 | fn main() {
    2 |     mut arr := [3, 2, 1]
    3 |     arr.sort(a < b, a)
      |         ~~~~~~~~~~~~~~
    4 | }
